packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,48 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isImmediatelyInvokedFunctionExpression(expressionPath) {
+    let expression = expressionPath.node.expression
```

Comment:
Oh, and install a linter in whatever text editor you're using.  In Atom, I have `linter-eslint`, and `linter-rubocop`.  Sublime Text has similar packages (which are super annoying to enable, which is why I switched to Atom.)